crates/core/src/network/reducers/init.rs

```diff
@@ -57,6 +58,8 @@ pub fn reduce_init(state: &mut NetworkState, root_state: &State, action_wrapper:
         p2p_config,
         Some(Address::from(network_settings.agent_id.clone())),
         Some(root_state.conductor_api.clone()),
+        // TODO: This probably shouldn't be none but I can't figure out how to get the tracer
+        None,
```

Comment:
I couldn't figure out how to get a tracer here @maackle It's needed to pass to the Sim2hWorker

Reply:
Well I thought we could get it in through the ActionWrapper. But then we would have to make Tracer `Serialize` :see_no_evil: